### DISCUSSION TITLE
How to connect Trino to Trino for federation

### DISCUSSION THREAD

**Discussion by SantoshKumar89** [trinodb/trino#22199] [Q&A]
Use case here is we have Three different regions where Trino is deployed. When a request goes to one of the Trino Instance, Trino should be able to connect to Trino's deployed in other region to search data. 

This is implemented in Starburst using Trino T2T Connector https://www.starburst.io/blog/query-federation-made-simple-at-comcast/ but I don't see any documentation around  T2T Connector.

**Comment by ebyhr** [comment]
Please take a look at https://github.com/trinodb/trino/issues/21791
